### PR TITLE
Continuing to fill the whitepaper

### PR DESCRIPTION
**Pull request by Lawouach**
More content for discussion :)

Signed-off-by: Sylvain Hellegouarch <sh@defuze.org>

### CODE REVIEW

File: WHITEPAPER.md [hunk 1]
```diff
@@ -130,14 +133,35 @@ it is good to check the basics of your system:
...
+* Monitoring: Your system constantly sends pulses about its current state. Those
+  signals tell you something about how it fares when you capture the right
+  metrics.
+* Automation: Whenever you deal with enough complexity, automation can save your
+  day. It is a wide topic and there are many levels of automation but starting
+  with Continuous Integration is a great starting point. Before moving on to
+  Continuous Delivery, Deployment and perhaps even GitOps.
+* Testing: You want to move fast but with a high degree of confidence. Testing
+  is a must-do practice to achieve it.
+
```
Comment (veggiemonk) [inline WHITEPAPER.md]: Maybe we can add alerting too ?

File: WHITEPAPER.md [hunk 2]
```diff
@@ -130,14 +133,35 @@ it is good to check the basics of your system:
...
+#### Do I need to get started in production?
+
+Production is relevant because it is more difficult to overfit the system to
+the purpose of the experiment, rendering the outcome more realistic, even if
+its reading is slightly more involved.
+
+While we may want this, starting in prod may not fit "getting started scenarios",
+and it is therefore, not only fine, but valuable to start your effort in your
+non-production environment until your team gets used to the practice.
```
Comment (veggiemonk) [inline WHITEPAPER.md]: Love it! 👏

File: WHITEPAPER.md [hunk 3]
```diff
@@ -151,18 +175,89 @@ The following phases may or may not be useful. I think it would be valuable if w
...
+a service it depends on.
+
+Common experiments can therefore:
+
+* add latency to a network call (either on the receive, send or both)
+* add jittering to create noise in the message
+* lose any number of packets along the channel
+* prevent name discovery
+* randomly close connections
+* inject random data into streams
```
Comment (veggiemonk) [inline WHITEPAPER.md]: fuzzing?

File: WHITEPAPER.md [hunk 4]
```diff
@@ -151,18 +175,89 @@ The following phases may or may not be useful. I think it would be valuable if w
...
+
 #### Vary Computing Resources
 
+Resources allocated to a service should not be trusted to always be available.
+Whether it's fighting for computing resources or attached devices going away,
+it is critical that you harness the impact of any of those situations on your
+system.
+
+* Reduce the available amount of CPU or memory to a service
+* Detach a device
```
Comment (veggiemonk) [inline WHITEPAPER.md]: I have little bit of trouble understanding what it means concretely to "detach a device".

File: WHITEPAPER.md [hunk 5]
```diff
@@ -151,18 +175,89 @@ The following phases may or may not be useful. I think it would be valuable if w
...
+is therefore highly valuable to a Chaos Engineer.
+
+Smulating data loss often depends on the model, architecture and storage of the
+system so it experimenting for it will take different shapes.
+
+* Remove storage
+* Change permissions so a service cannot read or write
+* Drop messages from a queue to prevent eventual consistency
+* Duplicate messages and deal with integrity
+* Perform a backup
```
Comment (veggiemonk) [inline WHITEPAPER.md]: * Restore a backup
* test database failover

File: WHITEPAPER.md (hunk 1 above)
Comment (Lawouach) [reply WHITEPAPER.md]: Indeed!

File: WHITEPAPER.md (hunk 4 above)
Comment (Lawouach) [reply WHITEPAPER.md]: Sorry for not being clear. I mean, remove a physical or virtual storage device for instance.

Comment (veggiemonk) [reply WHITEPAPER.md]: Oh that makes sense now. Thank you

File: WHITEPAPER.md (hunk 1 above)
Comment (mattouille) [reply WHITEPAPER.md]: I've seen the word GitOps passed around quite a bit but I fail to see how it's different from regular CI/CD; other than that it's strictly ops focused. If I'm being too nit-picky let me know it just seems like terms that are trying to find establishment probably don't have a place in a whitepaper.

**Pull request merged**